

IGTL_INIT_W: 0.707
IGTL_INIT_X: 0.0
IGTL_INIT_Y: 0.707
IGTL_INIT_Z: 0.0

resolution: 0.01

increase_path_x: 0.0
increase_path_y: 0.0
increase_path_z: 0.0

